lib/metrics/metrics.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+// Licensed under the Apache License, Version 2.0. See LICENSE
+
+export default {
```

Comment:
I see this one useful for people using expectations and stuff. What they need is just import our constants and use them for expectations config.
```ts
const METRICS = require('pwmetrics/lib/metrics-adapter/metrics');

module.exports = {
  url: 'http://my-awesome-site.com'
  flags: {
    expectations: true
  },
  expectations: {
    [METRICS. TTFCP]: {
       warn: '>=1000',
       error: '>=2000'
    }
  }
}
```

 Other then that cleaner for our code to have map metrics ids.